src/engines/fx/pingpong/pingpong.hpp

```diff
@@ -0,0 +1,62 @@
+#pragma once
+
+#include "core/audio/clock.hpp"
+#include "core/engine/engine.hpp"
+#include "core/input.hpp"
+#include "itc/itc.hpp"
+
+namespace otto::engines::pingpong {
+
+  using namespace core::props;
+
+  struct Screen;
+  struct Audio;
+
+  using Sender = core::engine::EngineSender<Audio, Screen>;
+
+  struct Props {
+    Sender sender;
+
+    /// Changes between free delay time and subdivisions
+    Sender::Prop<struct timetype_tag, bool> timetype = {sender, false};
+    /// Only changed when timetype is false.
+    Sender::Prop<struct free_time_tag, float> free_time = {sender, 0, limits(0, 1), step_size(0.01)};
+    /// Only changed when timetype is true
+    Sender::Prop<struct subdivision_tag, int> subdivision = {sender, 0, limits(0, 6)};
+    /// Actual delaytime in ms. Sent to the audio thread
+    // 6 seconds is the length of a measure at 40 bpm
+    Sender::Prop<struct free_time_tag, float> delaytime = {sender, 500, limits(0, 6)};
```

Comment:
Copy paste error?

Reply:
Yeah. I don't consider a 500 second delay time to be... _default_ behaviour.

Reply:
oh, i was refering to `struct free_time_tag`